data_validation/__main__.py

```diff
@@ -246,7 +246,7 @@ def build_config_managers_from_args(args):
 
     is_filesystem = source_client._source_type == "FileSystem"
     tables_list = cli_tools.get_tables_list(
-        args.tables_list, default_value=[], is_filesystem=is_filesystem
+        args.tables_list, default_value=[{}], is_filesystem=is_filesystem
```

Comment:
Is there a reason we need the default value to be a list with an empty dictionary? Can we just keep the default value as an empty list?

Reply:
when this piece of code is executed "for table_obj in tables_list:" for a default_value=[] it just does nothing as there is nothing to iterate and the return configs just returns an empty list. This was the original issue we had (no code gets executed). Having an empty dictionary at the least ensure the code "for table_obj in tables_list:" is iterated once and the code proceeds forward to run the custom queries listed in source and target files. 

Since its an empty dictionary, we assign table and schema to None in config_manager.py.